exercise/models.py

```diff
@@ -157,11 +157,19 @@ def to_account(self):
 
     @property
     def reveal_time(self):
-        email = ExerciseEmailProperties.objects.filter(
+        email_properties = ExerciseEmailProperties.objects.filter(
```

Comment:
While you're at it - can you please change this property method to use `self.exercise_specific_properties` instead of the filter?